.changeset/modern-berries-marry.md

```diff
@@ -0,0 +1,12 @@
+---
+"@comet/blocks-admin": minor
+---
+
+Add `label` prop to `ColumnsLayoutPreview`.
```

Comment:
```suggestion
Add `label` prop to `ColumnsLayoutPreview`

```

Slight improvements: The first line is the title and shouldn't have a period. Also, a new line is needed.